README.md

```diff
@@ -1 +1,74 @@
 # ecommerce-project
+
+This repository contains a complete ecommerce project built with React, Node, Express, Knex, and PostgreSQL. It is a great starting point for anyone looking to build a ecommerce application.
+
+## How its built
+
+This project features a robust ecommerce backend built with Express and Knex. It also includes a front-end built with React.
+
+- The front-end is built with React and React Router.
+- The backend is built with Node, Express, Knex, and PostgreSQL.
+- The project backend is deployed to Heroku.
+- The project front-end is deployed to Netlify.
```

Comment:
you can add the database details here, that we are using postgreSQL You can also mention the version of all the major tech we are using